iroh-gateway/src/client.rs

```diff
@@ -83,6 +89,7 @@ impl Client {
         &self,
```

Comment:
given that we don't call this anymore, should we just remove it?

Reply:
Intended to do that in https://github.com/n0-computer/iroh/issues/53, purely kept as a reference and testing point for the metrics pipe currently.